app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt

```diff
@@ -230,9 +184,10 @@ class RegistrationFragment : BaseFragment(), RegistrationView {
     override fun onDestroyView() {
         super.onDestroyView()
         presenter?.detachView()
+        _binding = null
     }
     private fun checkValidityOfPhoneNumber (isPhoneNumberValid: CountryCodePicker): Boolean {
```

Comment:
change `checkValidityOfPhoneNumber ` to `isPhoneNumberValid` and in the arguments change `isPhoneNumberValid` to `ccp` . This was a mistake that I should have rectified a long time back but since you are on it , could you please make the necessary changes ?

Reply:
yes @gururani-abhishek